static/sass/patterns/_icon-blocks.scss

```diff
@@ -0,0 +1,74 @@
+// Icon Blocks
+// ===========
+
+.icon-blocks-container {
+  @include center('medium-page');
+  @include grid;
+
+  h2 {
+    @supports not (display: grid) {
```

Comment:
With feature-queries, should we always have a fallback outside `@supports` for IE and older browsers? Or not worth it?